lib/convert.ts

```diff
@@ -1,7 +1,6 @@
-// @flow
-
 import * as ls from './languageclient';
-import {Point, Range} from 'atom';
+import {Point, ProjectFileEvent, Range, TextEditor} from 'atom';
+import {Diagnostic, DiagnosticType, TextEdit} from '../typings/atom-ide';
```

Comment:
Ideally we wouldn't import from `../typings/atom-ide` but instead from a "fake"  `atom-ide` module provided by a `@types/atom-ide` package.